erdpy/flows.py

```diff
@@ -1,8 +1,11 @@
 import logging
+
 from erdpy.projects import load_project
 from erdpy.environments import TestnetEnvironment
 from erdpy.accounts import Account
 from erdpy.contracts import SmartContract
+from erdpy.proxy.caller import ProxyApiCaller
```

Comment:
`from erdpy.proxy` import `ElrondProxy`? That is, renaming. Also, check out what a `__init__.py` files does :)

Reply:
done.